decidim-budgets/app/controllers/decidim/budgets/line_items_controller.rb

```diff
@@ -44,11 +44,11 @@ def destroy
       private
 
       def project
-        @project ||= Project.includes(:budget).find_by(id: params[:project_id], decidim_budgets_budget_id: params[:budget_id])
+        @project ||= Project.where(budget:).includes(:budget).find_by(id: params[:project_id])
```

Comment:
Or maybe just:
```suggestion
        @project ||= budget&.projects&.find_by(id: params[:project_id])
```

Is there some problem fetching the project through the budget record?